src/api/api.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const appId = 'aBI9ITaYCitPj6DreagZ';
+
+const getBooks = async () => axios.get(`${url}/apps/${appId}/books`).then((result) => {
+  const books = [];
+  if (result.status === 200) {
+    const { data } = result;
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
**Changes Required**
- [ ] Let us try to implement this nice piece of work without disabling the eslint restricted syntax. It's tough but that's what really makes code uniform and readable across teams collaborating on projects in the real industry.